autoscheduler/frontend/src/redux/actions/courseCards.ts

```diff
@@ -0,0 +1,81 @@
+import { ThunkAction } from 'redux-thunk';
+import fetch from '../testData';
+import { CourseCardOptions, SectionSelected } from '../../types/CourseCardOptions'; import {
+  AddCourseAction, ADD_COURSE_CARD, RemoveCourseAction, REMOVE_COURSE_CARD, UpdateCourseAction,
+  UPDATE_COURSE_CARD,
+} from '../reducers/courseCards'; import { RootState } from '../reducer'; import Meeting from '../../types/Meeting';
```

Comment:
```suggestion
} from '../reducers/courseCards'; 
import { RootState } from '../reducer'; 
import Meeting from '../../types/Meeting';
```

Don't think this was intentional, but these should be separated on different lines